### ISSUE TITLE
enable group/type filtering on /news page

### ISSUE THREAD

**Issue by skippy** [sjifire/website#62]
we'll want to be able to filter by post type.

**Comment by skippy** [comment]
related to, and perhaps supersedes #23

**Comment by icegulch** [comment]
Yeah, in 11ty, when paginating in the YML block, you have access to a before callback that allows you to filter a model based on whatever you need. that way, you can have /news, /advisories, etc all paginated. Does that sound like what we need?

**Comment by skippy** [comment]
I was thinking more as a dropdown so folks can see JUST the responder moments, or JUST the advisories... and then have pagination keep that filtering option of course.  

This is a v2, so lets keep it for later.

**Comment by icegulch** [comment]
you're right: dynamically changing the array would be hard -- beyond my paygrade? probably something that'd be possible with serverless?

however, i don't know that we really need to paginate (set a `size` limit) on the news landing page...even if we add little thumbnail images to the cards.

by not paginating / setting a size, filtering would be a cinch -- there'd be no need to dynamically change the `posts` array.

thoughts?
